src/hooks/use-one-ms-delayed-state.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback } from 'react';
+
+import { useDelayedState } from './use-delayed-state';
+
+/** Set a state with 1 ms delay */
+const DELAY_DURATION = 1;
+
+export const useOneMSDelayedState = <T>(
```

Comment:
Could you add a comment what the purpose of this hook is? I understand what it does - I just don't understand why I would want to use it...

Reply:
It is a wrapper for `useDelayedState`.
Purpose is for avoiding always this:
```ts
const [state, setState ] = useDelayedState(false)
...
// and somewhere usage, minimal delay 1ms:
setState(true, 1)
```
with this hook we will get:
```ts
const [state, setState ] = useOneMSDelayedState(false)
...
// and somewhere usage
setState(true)
```
so if we would have somewhere to replace, we just will replace `React.useState` with `useOneMSDelayedState` and done